File: BASICO/Reto_9/Solucion_1.py
word = input("Enter a word or phrase: ")
letter = input("Enter a letter: ")
verify = False
for c in word:
    if c.lower() == letter.lower():
        verify = True
if verify:
    print(letter, " Yes, it is")
else:
    print(letter, "No, it isn't")
    

File: BASICO/Reto_9/Solucion_1.cpp
#include <iostream>
#include <string>
using namespace std;
int main() {
    string word; char letter;
    bool verify = false;
    cout << "Enter a word or phrase: "; getline(cin, word);
    cout << "Enter a letter: "; cin >> letter;
    for (char c:word) {
        if (tolower(c) == tolower(letter)) {
            verify = true;
        }
    }
    if (verify) {
        cout << letter <<" yes, it is";
    } else {
        cout << letter << " no, it isn't";
    }
    return 0;
}